relationships/synthesis/EXT-SERVICE-to-INFRA-AWSSQSQUEUE.yml

```diff
@@ -1,13 +1,24 @@
 relationships:
-  - name: extServiceCallsInfraAwsSqsQueue
+  - name: extServiceCallsInfraAwsSqsQueue1_20
     version: "1"
     origins:
       - Distributed Tracing
+      - OpenTelemetry
     conditions:
       - attribute: eventType
         anyOf: [ "Span" ]
-      - attribute: rpc.service
-        anyOf: [ "SQS" ]
+      - attribute: messaging.system
+        anyOf: [ "aws.sqs" ]
+      - attribute: newrelic.source 
+        anyOf: [ "api.traces.otlp" ] 
+      - attribute: entity.type 
+        anyOf: [ "SERVICE" ] 
+      - attribute: aws.region
+        present: true
+      - attribute: aws.account.id
+        present: true
+      - attribute: messaging.destination
+        present: true 
```

Comment:
Does the [example Java application](https://source.datanerd.us/APM/entity-relationships-demo-apps/pull/23) you created emits these attributes? I do not see you manually adding them.